src/test/java/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAddCalculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
+    }
```

Comment:
```suggestion
    @NullAndEmptySource
    @ParameterizedTest
    public void splitAndSum_null_또는_빈문자(String input) {
        int result = StringAddCalculator.splitAndSum(input);
        assertThat(result).isEqualTo(0);
    }
```
NullAndEmptySource 애너테이션도 제공합니다. 

1단계에서 사용했던 ParameterizedTest를 활용하면 좋을 것 같네요
나머지 테스트 케이스도 ParameterizedTest로 변경할 수 있어보입니다.